### DISCUSSION TITLE
[Question] spring boot  yml configuration，The qos.enable configuration is not effective, but the qos.port configuration is effective

### DISCUSSION THREAD

**Discussion by pyshiweijia** [apache/dubbo#14165] [Question]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dubbo/issues?q=is%3Aissue) and found no similar issues.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Dubbo Version

Dubbo Java 3.0.6, OpenjDK 1.8, Ubuntu 20.04

### Steps to reproduce this issue

![1](https://github.com/apache/dubbo/assets/14867880/79003d57-1cee-4b73-9d98-9e753e0716bb)

### What you expected to happen

[DUBBO] qos won't be started because it is disabled. Please check dubbo.application.qos.enable is configured either in system property, dubbo.properties or XML/spring-boot configuration., dubbo version: 3.0.6

qos.enable=false&qos.port=33333&release=3.0.6&revision=3.0.6&service-name-mapping=true&side=provider&timestamp=1714385186196&version=1.0.0], dubbo version: 3.0.6, current host: 192.168.124.21

### Are you willing to submit a pull request to fix on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
Please test on the latest version
